src/components/ControlledForm/ControlledForm.stories.mdx

```diff
@@ -0,0 +1,107 @@
+# Form Component
+
+## Type: Simple
+
+## Form Props
+
+|     Prop      |                    Type                    |                                                                                                                                      Description                                                                                                                                       | Default | Mandatory |
+| :-----------: | :----------------------------------------: | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :-----: | :-------: |
+|    render     | (getInputControls, isValid, state) => void |            This props will render the Form fields. `getInputControls` will register the input and return and object with `onChange, value and, onValidationChange`. `isValid` will be true when the form is valid and `state` will be an object with all inputs and validation values. |    -    |   true    |
```

Comment:
@higorquintao let's give a bit more insight into what we expect regarding the `getInputControls` argument. It seems it expects a string as its own argument. But what does this string represent to the functionality of `getInputControls` and the `ControlledForm` component as a whole? It would also be helpful to have this additional info as an observation below this table. Please see the Dropdown or Input documentation as an example.